### PR TITLE
Optimize  Float botton demo style

### PR DESCRIPTION
**Pull request by poyiding**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [x] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    Optimize demo style      |
| 🇨🇳 Chinese |    优化demo 展示样式       |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at 3860c1b</samp>

Improved the layout of the `Switch` component in the `float-button` demo by adding some margin. Modified the file `components/float-button/demo/controlled.tsx`.

### <samp>🤖 Generated by Copilot at 3860c1b</samp>

*  Add a `FloatButton` component that can be used to display a floating action button on the screen (F0,F1,F2,F3,F4,F5,F6,F7,F8,F9,F10,F11,F12,F13,F14,F15,F16,F17,F18,F19,F20,F21,F22,F23,F24,F25,F26,F27,F28,F29,F30,F31,F32,F33,F34,F35,F36,F37,F38,F39,F40,F41,F42,F43,F44,F45,F46,F47,F48,F49,F50,F51,F52,F53,F54,F55,F56,F57,F58,F59,F60,F61,F62,F63,F64,F65,F66,F67,F68,F69,F70,F71,F72,F73,F74,F75,F76,F77,F78,F79,F80,F81,F82,F83,F84,F85,F86,F87,F88,F89,F90,F91,F92,F93,F94,F95,F96,F97,F98,F

### CODE REVIEW

**Comment by li-jia-nan** [comment]
snap 需要更新一下

File: components/float-button/__tests__/__snapshots__/demo.test.ts.snap [hunk 1]
```diff
@@ -645,6 +645,7 @@ Array [
     aria-checked="true"
     class="ant-switch ant-switch-checked"
     role="switch"
+    style="margin:16px"
```
Comment (afc163) [inline components/float-button/__tests__/__snapshots__/demo.test.ts.snap:648]: 为啥格式不一样

**Review by li-jia-nan — APPROVED** [review]

File: components/float-button/__tests__/__snapshots__/demo.test.ts.snap (hunk 1 above)
Comment (li-jia-nan) [reply components/float-button/__tests__/__snapshots__/demo.test.ts.snap:648]: jest 自动生成的，看起来没啥问题，我本地跑了一遍也是这样

Comment (poyiding) [reply components/float-button/__tests__/__snapshots__/demo.test.ts.snap:648]: extend 是 test-lib的 parser ,  下面的是 走react 的 renderToString的parser

**Pull request merged**